docs_src/microservices/application/ApplicationFunctionsSDK.md

```diff
@@ -131,7 +131,9 @@ E.G. NewFilter([] {"Device1", "Device2"}).FilterByDeviceName
 
 ### Filtering
 
-There are two basic types of filtering included in the SDK to add to your pipeline. Theses provided Filter functions return a type of events.Model. If filtering results in no remaining data, the pipeline execution for that pass is terminated. If no values are provided for filtering, then data flows through unfiltered.
+There are two basic types of filtering included in the SDK to add to your pipeline. There is also an option to `Filter Out` specific items. Theses provided Filter functions return a type of events.Model. If filtering results in no remaining data, the pipeline execution for that pass is terminated. If no values are provided for filtering, then data flows through unfiltered.
+
+- `filterout` - This parameter will determine how the filter will filter out the items. If set to true all items found in the filter will be removed. If set to false all items found in the filter will be returned. If filterValues is empty then all items will be returned.
```

Comment:
Too far up now ;-). It is part of NewFilter().  Please move down just above - `FilterByDeviceName`